node/src/components/consensus/highway_core/highway.rs

```diff
@@ -429,6 +423,17 @@ impl<C: Context> Highway<C> {
         .unwrap_or_default()
     }
 
+    fn on_new_evidence(
+        &mut self,
+        evidence: &Evidence<C>,
+        rng: &mut dyn CryptoRngCore,
+    ) -> Vec<Effect<C>> {
+        // TODO: Add `InstanceId` to endorsements.
```

Comment:
Maybe that's actually not necessary, since the vote has an instance ID, so the vote's hash depends on it. So you can't use endorsements from other instances for votes in this one.

Reply:
Good point.